tune_the_model/cli.py

```diff
@@ -15,6 +15,10 @@
     TuneTheModelException,
 )
 
+import logging

```

Comment:
I think that we should move this import to other system ones like os and json modules. We ll add a linter check soon.